src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -248,6 +264,11 @@ module CommandResponse =
 
   let info (serialize : Serializer) (s: string) = serialize { Kind = "info"; Data = s }
   let error (serialize : Serializer) (s: string) = serialize { Kind = "error"; Data = s }
```

Comment:
Sure, np for me. i just didnt want to add work to you all, because workaround (maintain old contract for generic) was easy :D